js/src/sdk/models/actions.ts

```diff
@@ -243,13 +243,17 @@ export class Actions {
         apps: Array<string>,
         useCase: string,
         limit?: number,
+        filterByAvailableApps?: boolean
     }): Promise<Array<string>> {
         try {
             const { data: res } = await apiClient.actionsV2.advancedUseCaseSearch({
                 query: {
                     apps: data.apps.join(","),
+                    limit: data.limit || undefined,
+                    filterByAvailableApps: data.filterByAvailableApps || false
```

Comment:
The `filterByAvailableApps` parameter is added but defaults to `false` in the implementation. This should be documented in the type definition to make the default behavior clear to API consumers.